src/plugins/expressions/public/loader.ts

```diff
@@ -67,8 +69,8 @@ export class ExpressionLoader {
       }
     });
 
-    this.data$.subscribe((data) => {
-      this.render(data);
+    this.data$.subscribe(({ result }) => {
+      this.render(result);
```

Comment:
loader needs to receive an option if partial results should be rendered or not,

if option is false, we only render when partial===false